docs/examples.rst

```diff
@@ -213,6 +216,134 @@ write can be performed by::
 This will write the rootfs image ``rootfs.ext4`` to the slot ``rootfs.0``. Note
 that this bypasses all update mechanics like hooks, slot status etc.
 
+Example Slot Configurations
+---------------------------
+
+This provides some common examples on how to configure slots in your
+system.conf for different scenarios.
+
+Symmetric A/B Setup
+~~~~~~~~~~~~~~~~~~~
+
+This is the default case when having a fully-redundant root file system
+
+.. code-block:: cfg
+  :emphasize-lines: 3, 6, 8, 11
+
+  [...]
+
+  [slot.rootfs.0]
+  device=/dev/sda2
+  type=ext4
+  bootname=A
+
+  [slot.rootfs.1]
+  device=/dev/sda3
+  type=ext4
+  bootname=B
+
+
+Asymmetric A/Recovery Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case storage is too restricted for a full A/B redundancy setup, an
+asymmetric setup with a dedicated update / recovery slot can be used.
+This slot can be way smaller than the rootfs one as it needs to contain only
+the tools for updating the rootfs slot.
+
+.. code-block:: cfg
+  :emphasize-lines: 3, 6, 8, 11
+
+  [...]
+
+  [slot.recovery.0]
+  device=/dev/sda2
+  type=ext4
+  bootname=A
+
+  [slot.rootfs.0]
+  device=/dev/sda3
+  type=ext4
+  bootname=B
```

Comment:
What about s/bootname=B/bootname=R/ instead?
And what about adding `readonly=true` and some explanation for this choice in the preceding paragraph?

Reply:
Good hint!